src/Common/src/System/Net/Logging/Logging.cs

```diff
@@ -0,0 +1,724 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Threading;
+
+namespace System.Net
+{
+    internal class Logging
+    {
+        private static volatile bool s_LoggingEnabled = true;
```

Comment:
@stephentoub This isn't yet ready but we need the class to compile. Long term ETW tracing will always be enabled similarly to how GlobalLog works now.